_posts/2017-12-30-eithersch.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+author: kiralybalint
+date: 2017-12-30 21:01:00 CET
+title: "EitherSCH"
+comment: true
+---
+
+Nincs programod szilveszterre? Írtunk neked egyet..
+
+Természetesen a komoly fejlesztés mellett szükség van néha a kikapcsolódásra, ezért megrendezésre került a Kir-Dev téli tábor. Persze az igazi fejlesztő nem bírja ki fejlesztés nélkül 1 napnál tovább így a második este kissé iluminált állapotban neki áltunk kódolni. A cél az volt, hogy az either.io-hoz hasonló játékot hozzunk létre saját SCH-s kérdésekkel. 
```

Comment:
iluminált => illuminált

Kérlek a helyesírás ellenőrzőt kapcsoljátok be mikor blog bejegyzést írtok.

Másrészt tartalmi dolog:

> Persze az igazi fejlesztő nem bírja ki fejlesztés nélkül 1 napnál tovább

Én ezt átfogalmaznám úgy, hogy ne általánosítson, ne a "brogrammer" kultúrát erősítse és úgy általában ne legyen flegma. Mondjuk például:

A körtagok nem bírják kódolás nélkül, ezért a második este ...